src/Shared/Recruit.Vacancies.Client/Infrastructure/ReferenceData/ApprenticeshipProgrammes/ApprenticeshipProgrammeProvider.cs

```diff
@@ -40,29 +40,23 @@ public async Task<ApprenticeshipStandard> GetApprenticeshipStandardVacancyPrevie
             return await _outerApiClient.Get<GetVacancyPreviewApiResponse>(new GetVacancyPreviewApiRequest(programmedId));
         }
 
-        public async Task<IEnumerable<IApprenticeshipProgramme>> GetApprenticeshipProgrammesAsync(bool includeExpired = false)
+        public async Task<IEnumerable<IApprenticeshipProgramme>> GetApprenticeshipProgrammesAsync(bool includeExpired = false, int? ukprn = null)
         {
-            var queryItem = await GetApprenticeshipProgrammes();
+            var queryItem = await GetApprenticeshipProgrammes(ukprn);
             return includeExpired ? 
                 queryItem.Data : 
                 queryItem.Data.Where(x =>x.IsActive || (x.ApprenticeshipType == TrainingType.Foundation && IsStandardActive(x.EffectiveTo,x.LastDateStarts)));
         }
 
-        private Task<ApprenticeshipProgrammes> GetApprenticeshipProgrammes()
+        private async Task<ApprenticeshipProgrammes> GetApprenticeshipProgrammes(int? ukprn)
         {
             var includeFoundationApprenticeships = _feature.IsFeatureEnabled(FeatureNames.FoundationApprenticeships);
 
-            return _cache.CacheAsideAsync(CacheKeys.ApprenticeshipProgrammes,
-                _timeProvider.NextDay6am,
```

Comment:
I have removed the caching. Although the list of available courses is still not going to be changed often, given that we now link to the Manage Standards page from the add training to a vacancy page to encourage the Provider to edit their listed offering if need be, it won't make sense to the user if they return to the vacancy training page and can't immediately see the edits they just made.